### ISSUE TITLE
Extract PSK candidates from captures

### ISSUE THREAD

**Issue by RealEnder** [RealEnder/dwpa#42]
hcxpcaptool -E -U -I and try those before returning work to the client

**Comment by ZerBea** [comment]
Great idea. Many, many PSKs can be found in the traffic.
A good idea would be:
```
hcxpcapngtool -E -I-U to get all possible candidates
hcxeiutool -s on hcxpcapngtool -E -I -U output to get a list with raw candidates and add to the list
send list from hcxeiutool -s and rules (luc: + 0-99, 1950-2011) to python client (help_crack.py) and  let him do the work load
```

**Comment by RealEnder** [comment]
Fixed in da31067ba20294f9baf4d99bbad3ae1baf86f64f
